ios/FluentUI/BottomSheet/BottomSheetViewController.swift

```diff
@@ -0,0 +1,404 @@
+@objc(MSFBottomSheetViewController)
+public class BottomSheetViewController: UIViewController {
+    @objc public weak var delegate: BottomSheetViewControllerDelegate?
+    /// BottomSheetViewController's view height in collapsed mode.
+    @objc public var collapsedHeight: CGFloat = Constants.collapsedHeight {
+        didSet {
+            updateFrame()
+        }
+    }
+
+    /// BottomSheetViewController's view height in expanded mode. If the value is -1, it will grow to the its windows safe area height.
+    @objc public var preferredExpandedHeight: CGFloat = -1 {
+        didSet {
+            updateFrame()
+        }
+    }
+
+    /// BottomSheetViewController's view width in iPad when horizontal size is regular
+    @objc public var preferredContentWidthForPad: CGFloat = Constants.contentWidthForPad {
+        didSet {
+            updateFrame()
+        }
+    }
+
+    @objc public var showResizeHandle: Bool = true {
+        didSet {
+            if oldValue != showResizeHandle {
+                resizingHandleView.isHidden = !showResizeHandle
+            }
+        }
+    }
+
+    private var animator: UIViewPropertyAnimator?
+    private var contentViewController: UIViewController
+    private var currentState: BottomSheetViewState = .collapse
+    private let containerView: UIStackView = {
+        let view = UIStackView()
+        view.axis = .vertical
+        return view
+    }()
+
+    private lazy var dimmingView: DimmingView = {
+        let view = DimmingView(type: .black)
+        view.isUserInteractionEnabled = false
+        return view
+    }()
+
+    private lazy var panGestureRecognizer: UIPanGestureRecognizer = {
+        let recognizer = UIPanGestureRecognizer()
+        recognizer.addTarget(self, action: #selector(handlePan))
+        return recognizer
+    }()
+
+    private lazy var resizingHandleView: ResizingHandleView = {
+        let resizingHandleView = ResizingHandleView()
+        resizingHandleView.isAccessibilityElement = true
+        resizingHandleView.accessibilityTraits = .button
+        resizingHandleView.isUserInteractionEnabled = true
+        resizingHandleView.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(handleResizingHandleViewTap)))
+        return resizingHandleView
+    }()
+
+    @objc public init(with contentViewController: UIViewController) {
+        self.contentViewController = contentViewController
+        super.init(nibName: nil, bundle: nil)
+        addChild(contentViewController)
+        contentViewController.didMove(toParent: self)
+    }
+
+    public required init?(coder aDecoder: NSCoder) {
+        preconditionFailure("init(coder:) has not been implemented")
+    }
+
+// MARK: - Lifecycle methods
+
+    open override func viewDidDisappear(_ animated: Bool) {
+        removeDimmingView()
+        super.viewDidDisappear(animated)
+    }
+
+    open override func viewDidLoad() {
```

Comment:
This looks a bit more like a loadView than a viewDidLoad? viewDidLoad is additional configuration after setting up the hierarchy (usually from a xib), this just looks like we're setting up the hierarchy